WooCommerce/Classes/ViewRelated/Coupons/CouponDetails/CouponDetails.swift

```diff
@@ -256,11 +256,23 @@ struct CouponDetails: View {
                 Text(String.localizedStringWithFormat(Localization.maximumSpend, viewModel.maximumAmount))
                     .renderedIf(viewModel.maximumAmount.isNotEmpty)
 
+                Text(String.localizedStringWithFormat(Localization.singularUsageLimitPerCoupon, viewModel.usageLimit))
+                    .renderedIf(viewModel.usageLimit == 1)
+
+                Text(String.localizedStringWithFormat(Localization.pluralUsageLimitPerCoupon, viewModel.usageLimit))
+                    .renderedIf(viewModel.usageLimit > 1)
+
                 Text(String.localizedStringWithFormat(Localization.singularLimitPerUser, viewModel.usageLimitPerUser))
                     .renderedIf(viewModel.usageLimitPerUser == 1)
 
                 Text(String.localizedStringWithFormat(Localization.pluralLimitPerUser, viewModel.usageLimitPerUser))
                     .renderedIf(viewModel.usageLimitPerUser > 1)
+
+                Text(String.localizedStringWithFormat(Localization.singularItemsInCartUsageLimit, viewModel.limitUsageToXItems))
+                    .renderedIf(viewModel.limitUsageToXItems == 1)
+
+                Text(String.localizedStringWithFormat(Localization.pluralItemsInCartUsageLimit, viewModel.limitUsageToXItems))
+                    .renderedIf(viewModel.limitUsageToXItems > 1)
```

Comment:
We can also simplify this following the suggestion for `usageLimit` above.

Reply:
Sure! Done in https://github.com/woocommerce/woocommerce-ios/commit/1e0a4060d199e0ece9bf9353347bdd4b02ff485d and https://github.com/woocommerce/woocommerce-ios/commit/a312d0fb86d7fb9ccdfed2bb78210c557e908766.

I also took the liberty and adapted the `usageLimitPerUser` to also take advantage of the new pluralization function.

Reply:
Great, thanks!